doc/source/default-settings.rst

```diff
@@ -743,7 +742,7 @@ Open options
 
         mark-opener-format = zathura -P {mark[value]}
 
-Serve (Web App) options
+Serve (web app) options
```

Comment:
Explanation for changes like this: almost all of our headers/titles use textcase like

"Title foo bar baz"

instead of

"Title Foo Bar Baz"

So I made the few outliers consistent with the rest.